test/unit/controllers/StatusControllerSpec.scala

```diff
@@ -102,10 +102,7 @@ class StatusControllerSpec extends UnitSpec
 
   private val errorResultBadgeIdentifier = errorBadRequest("X-Badge-Identifier header is missing or invalid").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
   private val errorResultMissingIdentifiers = errorBadRequest("Both X-Submitter-Identifier and X-Badge-Identifier headers are missing").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
-  private val errorResultInvalidMrn = errorBadRequest("Invalid MRN").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
-  private val errorResultInvalidDucr = errorBadRequest("Invalid DUCR").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
-  private val errorResultInvalidUcr = errorBadRequest("Invalid UCR").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
-  private val errorResultInvalidInventoryReference = errorBadRequest("Invalid InventoryReference").XmlResult.withHeaders(X_CONVERSATION_ID_HEADER)
```

Comment:
Will these not be useful when a user sends one of these search parameters (non-empty content) but the values are invalid?

Reply:
Programme doesn't want that text in any error msg.